agent/cache/testing.go

```diff
@@ -60,7 +60,7 @@ func TestCacheNotifyChResult(t testing.T, ch <-chan UpdateEvent, expected ...Upd
 	}
 
 	got := make([]UpdateEvent, 0, expectLen)
-	timeoutCh := time.After(50 * time.Millisecond)
+	timeoutCh := time.After(75 * time.Millisecond)
```

Comment:
The `t.Fatalf` below says 50ms still, I guess we should update it. I think the message could say "timeout while waiting for a result".

Reply:
Changed message